### PR TITLE
Update jobs to be more unified, fix & add more tests

### PR DESCRIPTION
**Pull request by Yamsha75**

### CODE REVIEW

File: jobs/morele.py [hunk 1]
```diff
@@ -1,33 +1,58 @@
...
-    title = product_link.text
-    offer_url = product_link["href"]
-    image_url = morele_soup.find("a", {"class": "prom-box-image"}).img["src"]
+    old_price = soup.find("div", {"class": "promo-box-old-price"}).get_text(strip=True)
+    new_price = soup.find("div", {"class": "promo-box-new-price"}).get_text(strip=True)
 
-    old_price = morele_soup.find("div", {"class": "promo-box-old-price"}).text
-    new_price = morele_soup.find("div", {"class": "promo-box-new-price"}).text
-    coupon = morele_soup.find("div", {"class": "promo-box-code-value"}).text
+    coupon = soup.find("div", {"class": "promo-box-code-value"}).text
```
Comment (Yamsha75) [inline jobs/morele.py:45]: ".text" should be swapped for ".get_text(strip=True)" like lines 42-43

File: jobs/base.py [hunk 2]
```diff
@@ -1,4 +1,26 @@
...
 Offer = namedtuple("Offer", ["title", "description", "offer_url", "image_url"])
+
+
+def parse_price(price: str):
+    price.replace(",", ".")
+    price = match(r"(\d+(\.\d+)?).*", price).group(1)
+    return float(price)
+
+
+def prepare_description(old_price: str, new_price: str, bottom_text: str = None):
```
Comment (dzkb) [inline jobs/base.py:14]: COOL MEME

TEKST DOLNY

File: listener.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+from apscheduler.events import EVENT_JOB_EXECUTED, JobExecutionEvent
+from apscheduler.schedulers.base import BaseScheduler
+
+
+class JobMisfireError(RuntimeError):
+    pass
+
+
+def add_listener(
```
Comment (dzkb) [inline listener.py:12]: I'd suggest renaming the whole module and function to indicate the purpose: e.g. retry.py

File: tests/conftest.py [hunk 4]
```diff
@@ -3,19 +3,20 @@
 
 @pytest.fixture
 def xkom_hotshot_html():
```
Comment (dzkb) [inline tests/conftest.py:5]: It's no longer HTML

**Review by dzkb — COMMENTED** [review]
Please format code with black & isort.

**Review by dzkb — APPROVED** [review]

**Pull request merged**